src/app/view/editor-edit-tools-view-component/global-nodes-management/global-nodes-management.component.html

```diff
@@ -0,0 +1,52 @@
+<section class="global-nodes-management">
+  <div class="search">
+    <input
+      sbbInput
+      type="search"
+      [(ngModel)]="query"
+      [placeholder]="
+        'app.view.editor-edit-tools-view-component.nodes-search-placeholder' | translate
+      "
+    />
+  </div>
+  <div class="nodes-list">
+    @if (getVisibleNodes().length) {
```

Comment:
Hum I've not worked with Angular for a while, but why not use a useMemo for `visibleNodes` ?